src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/invalidjavadocposition/InputInvalidJavadocPosition.java

```diff
@@ -0,0 +1,47 @@
+package /** violation */ com.puppycrawl.tools.checkstyle.checks.javadoc.invalidjavadocposition;
+
+/** violation */
+import java.lang.String;
+
+/** violation **/
+/** valid **/
+class InputInvalidJavadocPosition {
+    /** violation **/    
+}
+/* ignore */
+class InputInvalidJavadocPosition2 {
+    /** violation */
+    /** valid */
+    int field1;
+    /** valid */
+    int[] field2;
+    /** valid */
+    public int[] field3;
+    /** valid */
+    @Deprecated int field4;
+
+    int /** violation */ field20;
+    int field21 /** violation */;
+    @Deprecated /** violation */ int field22;
+
+    void method1() {}
+    /** valid */
+    void method2() {}
+    /** valid */
+    <T> T method3() { return null; }
+    /** valid */
+    String[] method4() { return null; }
+
+    void /** violation */ method20() {}
+    void method21 /** violation */ () {}
+    void method22(/** violation */) {}
+    void method23() /** violation */ {}
+    void method24() { /** violation */ }
+    void method25() { /** violation */ int variable; }
```

Comment:
Found by IT's warn double checker.
A violation isn't being produced here. It thinks the variable is a field.

Reply:
This has not been fixed. It is outside the code of this PR, so it will have to be another issue/PR.